docs/api/mock.md

```diff
@@ -7,7 +7,7 @@ import { vi } from 'vitest'
 
 const fn = vi.fn()
 fn('hello world')
-fn.mock.calls[0] === ['hello world']
+fn.mock.calls[0][0] === 'hello world'
```

Comment:
It's important to _show_ the array. If this is a confusing representation, I propose


```suggestion
fn.mock.calls[0] // == ['hello world']
```

Reply:
@sheremet-va I think the best way would be to change these examples to use real expect assertions:
```ts
epect(fn.mock.calls[0]).toEqual(['hello world'])
```
as done in most other examples in the documentation. If you agree, I would adapt similar examples in the docs as well.

Reply:
we don't want to introduce different concepts in this part of the guide, like assertions

Reply:
Ok, then I would say it's best to leave it as it is.